packages/ember/typescript/src/ensure-ts-transpilation.js

```diff
@@ -120,4 +122,15 @@ function buildUtils(root, j) {
   };
 }
 
-export async function enableTSInAddonIndex() {}
+export async function enableTSInAddonIndex() {
+  // TODO: add the config to the index.js file: https://github.com/emberjs/ember-cli-babel#enabling-typescript-transpilation
```

Comment:
I'm not very familiar with jscodeshift, so I just display a warning for now. I'll try to implement it when I find some more time.

Reply:
no worries, I whipped one up for ya:

https://astexplorer.net/#/gist/813a0daa02c4cd9dc8f69f677cd09786/012b9928d8cbd687eb0c81948f5fae18dd15adb6

Reply:
Thanks!